hickle v5.0.2

telegraphic opened the pull request:
Attempt to fix https://github.com/telegraphic/hickle/issues/168

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

telegraphic committed:
Update meta.yaml

Attempt to fix https://github.com/telegraphic/hickle/issues/168

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

It appears you are making a pull request from a branch in your feedstock and not a fork. This procedure will generate a separate build for each push to the branch and is thus not allowed. See our [documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests) for more details.

Please close this pull request and remake it from a fork of this feedstock.

Have a great day!

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * noarch: python recipes are required to have a lower bound on the python version. Typically this means putting `python >=3.6` in **both** `host` and `run` but you should check upstream for the package's Python compatibility.

telegraphic committed:
Update meta.yaml

Addressing https://github.com/conda-forge/hickle-feedstock/pull/13#issuecomment-1337546343

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * ``requirements: run: numpy>=1.8,!=1.20`` must contain a space between the name and the pin, i.e. ``numpy >=1.8,!=1.20``
 * ``requirements: run: h5py>=2.10.0`` must contain a space between the name and the pin, i.e. ``h5py >=2.10.0``

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

It appears you are making a pull request from a branch in your feedstock and not a fork. This procedure will generate a separate build for each push to the branch and is thus not allowed. See our [documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests) for more details.

Please close this pull request and remake it from a fork of this feedstock.

Have a great day!

telegraphic committed:
Update meta.yaml

Adding space so linter doesn't freak out

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

It appears you are making a pull request from a branch in your feedstock and not a fork. This procedure will generate a separate build for each push to the branch and is thus not allowed. See our [documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests) for more details.

Please close this pull request and remake it from a fork of this feedstock.

Have a great day!

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

It appears you are making a pull request from a branch in your feedstock and not a fork. This procedure will generate a separate build for each push to the branch and is thus not allowed. See our [documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests) for more details.

Please close this pull request and remake it from a fork of this feedstock.

Have a great day!

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

It appears you are making a pull request from a branch in your feedstock and not a fork. This procedure will generate a separate build for each push to the branch and is thus not allowed. See our [documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests) for more details.

Please close this pull request and remake it from a fork of this feedstock.

Have a great day!

telegraphic commented:
@conda-forge-admin, please update circle

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

It appears you are making a pull request from a branch in your feedstock and not a fork. This procedure will generate a separate build for each push to the branch and is thus not allowed. See our [documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests) for more details.

Please close this pull request and remake it from a fork of this feedstock.

Have a great day!

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I just wanted to let you know that I updated the circle-ci
deploy key and followed the project.

telegraphic commented:
*sigh*
